lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php

```diff
@@ -1888,10 +1551,7 @@ function isTrue()
  */
 function isType($type)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$type". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)